Tasks/AzureKeyVault/operations/KeyVault.ts

```diff
@@ -162,4 +176,20 @@ export class KeyVault {
 
         return error;
     }
+
+    private generateAndUploadProvisionKeyVaultPermissionsScript(): void {
+        let tempPath = tl.getVariable('Agent.BuildDirectory') || tl.getVariable('Agent.ReleaseDirectory') || process.cwd();
```

Comment:
What does this mean? It will prefer agent.ReleaseDirectory over pwd?

Reply:
yes